imports/client/components/ProfileList.tsx

```diff
@@ -367,6 +384,79 @@ const ProfileList = ({
     );
   }, [hunt, canSyncDiscord, syncDiscord]);
 
+  const invitationLink = useMemo(() => {
+    if (!hunt || !canInvite || !invitationCode) {
+      return null;
+    }
+
+    const copyTooltip = <Tooltip>Copy to clipboard</Tooltip>;
+
+    const invitationUrl = `${window.location.origin}/join/${invitationCode}`;
```

Comment:
Small nit: I think this is slightly better expressed as
```ts
Meteor.absoluteUrl(`/join/${invitationCode}`)
```

Reply:
Done, thanks!